.github/workflows/codeql-analysis.yml

```diff
@@ -20,7 +24,6 @@ jobs:
   analyze:
     name: Analyze
     runs-on: ubuntu-latest
-    if: github.event_name == 'schedule' || github.event.pull_request.user.login == 'dependabot[bot]'
```

Comment:
Nice! I'm not sure why I came up with that weird "if" condition when `paths` were right there :-)